manage/import-export.js

```diff
@@ -51,6 +55,40 @@ onDOMready().then(() => {
   });
 });
 
+function showTimeToBackupTooltip() {
+  const wrap = $('#backup');
+  if (wrap) {
+    $('#backup').open = true;
+    wrap.classList.add('show-reminder');
+  }
+}
+
+function setLastBackup({ update = true }) {
+  const wrap = $('#backup');
+  if (update) {
+    localStorage['stylus-last-backup'] = Date.now();
+  }
+  if (wrap) {
+    wrap.classList.remove('show-reminder');
+  }
+  if (!prefs.get('manage.backup.expanded')) {
+    $('#backup').removeAttribute('open');
```

Comment:
I think we can remove this behavior. Collapsing a section when clicking on a notification is confusing IMHO.